cmd/kubeadm/app/master/manifests.go

```diff
@@ -459,3 +452,22 @@ func getSelfHostedAPIServerEnv() []api.EnvVar {
 
 	return append(getProxyEnvVars(), podIPEnvVar)
 }
+
+func getAuthzParameters(authzMode string) []string {
+	command := []string{}
+	// RBAC is always on. If the user specified
+	authzModes := []string{authzmodes.ModeRBAC}
+	if len(authzMode) != 0 && authzMode != authzmodes.ModeRBAC {
```

Comment:
Do we want to interpret authzModes as a list similar to the way the apiserver does?  That would change this code quite a bit.  But it might be good to mirror what the apiserver does.